source/platform/spongeapi.rst

```diff
@@ -2,11 +2,108 @@
 SpongeAPI 
 =========
 
-Adventure targets SpongeAPI 7.
+Adventure provides a platform for SpongeAPI 7 for *Minecraft: Java Edition* 1.12. For SpongeAPI 8 and up (targeting *Minecraft: Java Edition* 1.16.4), Adventure is the native text library, so no platform is needed.
 
+To get started using this platform, add the artifact to your build file:
 
-Accessing through Guice
------------------------
+First, add the repository:
 
+.. tabs::
+   
+   .. group-tab:: Maven
 
-``SpongePlatform`` can be injected by Guice!
\ No newline at end of file
+      .. code:: xml
+
+         <repositories>
+             <!-- ... -->
+             <repository> <!-- for development builds -->
+               <id>sonatype-oss</id>
+               <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+             </repository>
+             <!-- ... -->
+         </repositories>
+   
+   .. group-tab:: Gradle (Groovy)
+
+      .. code:: groovy
+
+         repositories {
+            // for development builds
+            maven {
+                name = 'sonatype-oss'
```

Comment:
```suggestion
                name = "sonatype-oss-snapshots"
```
We should be consistent with quotes in Gradle buildscripts, and stick to `"` I think.